src/views/Books/Books.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import BooksComp from '../../components/Books/BooksComp';
+import { fetchBooks } from '../../services/books';
+
+export default function Books() {
+  const [error, setError] = useState('');
+  const [books, setBooks] = useState([]);
+  useEffect(() => {
+    try {
+      const fetchData = async () => {
+        const data = await fetchBooks();
+        setBooks(data);
+      };
+      fetchData();
+    } catch (e) {
+      setError(e);
+    }
```

Comment:
Excellent work using error handling within your useEffect, it is super important with 3rd party APIs.